cbmc/proofs/Makefile.common

```diff
@@ -102,7 +102,7 @@ CBMCFLAGS += \
 goto: $(ENTRY).goto
 
 cbmc.txt: $(ENTRY).goto
-	- cbmc $(CBMCFLAGS) --trace $< >$@ 2>&1
+	- cbmc $(CBMCFLAGS) --trace $< 2>&1 | tee $@
```

Comment:
Don't make this change.  We don't want the tee in the Makefile, it is Linux specific and it masks errors (the return code is set by tee and not the run of cbmc).  Anyone who wants to monitor cbmc can to a tail -f cbmc.txt in another shell.